examples/cross-windows/flake.nix

```diff
@@ -34,12 +34,12 @@
 
           depsBuildBuild = with pkgs; [
             pkgsCross.mingwW64.stdenv.cc
-            pkgsCross.mingwW64.windows.pthreads
           ];
 
           nativeBuildInputs = with pkgs; [
             # We need Wine to run tests:
             wineWowPackages.stable
+            pkgsCross.mingwW64.windows.pthreads
```

Comment:
I think to make it equivalent, it should go to buildInputs. Did that not work for you?